### PR TITLE
Show dialog explicit

### PR DESCRIPTION
**Pull request by jreidinger**
## Problem

openqa needs a window to collect logs before reboot even in auto modes

## Solution

Respect explicit reboot timeout on command line even in automatic modes.

## Testing

- *Added a new unit test*
- *Tested manually*

## Screenshots

*If the fix affects the UI attach some screenshots here.*

### CODE REVIEW

File: src/lib/installation/clients/inst_finish.rb [hunk 1]
```diff
@@ -148,6 +147,9 @@ def report_messages
       if Linuxrc.reboot_timeout
         Report.DisplayMessages(true, Linuxrc.reboot_timeout)
       else
+        # Skip in autoinstallation and autoupgrade mode only if not explicitelly
+        # set on command line (bsc#1231522)
+        return if Mode.auto
```
Comment (teclator) [inline src/lib/installation/clients/inst_finish.rb:152]:
```suggestion
        return if Mode.auto
        
```

**Review by teclator — APPROVED** [review]
LGTM

File: src/lib/installation/clients/inst_finish.rb [hunk 2]
```diff
@@ -148,6 +147,10 @@ def report_messages
       if Linuxrc.reboot_timeout
         Report.DisplayMessages(true, Linuxrc.reboot_timeout)
       else
+        # Skip in autoinstallation and autoupgrade mode only if not explicitelly
```
Comment (teclator) [inline src/lib/installation/clients/inst_finish.rb:150]:
```suggestion
        # Skip in autoinstallation and autoupgrade mode only if not explicitly
```

Comment (jreidinger) [reply src/lib/installation/clients/inst_finish.rb]: yeah, my spell checker catched it in changes, but here it is not used.

**Pull request merged**